gitlab/v4/objects.py

```diff
@@ -2641,6 +2637,43 @@ class ProjectAccessRequestManager(ListMixin, CreateMixin, DeleteMixin,
     _from_parent_attrs = {'project_id': 'id'}
 
 
+class ProjectMergeRequestApproval(SaveMixin, RESTObject):
+    _id_attr = None
+
+
+class ProjectMergeRequestApprovalManager(GetWithoutIdMixin, UpdateMixin,
+                                         RESTManager):
+    _path = '/projects/%(project_id)s/approvals'
```

Comment:
Shouldn't the url be `/projects/%(project_id)s/merge_requests/%(merge_request_iid)s/approvals`? I think you made a mix of projects approvals and merge requests approvals (https://docs.gitlab.com/ee/api/merge_request_approvals.html#project-level-mr-approvals and https://docs.gitlab.com/ee/api/merge_request_approvals.html#merge-request-level-mr-approvals).

You could probably implement both endpoints BTW :)

Reply:
Sure thing !
I renamed the classes `ProjectApproval`, `ProjectApprovalManager` and the attribute `approvals` for consistency.